InteractionBase/src/handlers/GenericHandler.cpp

```diff
@@ -424,6 +425,8 @@ void GenericHandler::keyPressEvent(Visualization::Item *target, QKeyEvent *event
 			// Check if the parent of the node is a list and if so, delete this node
 			if (auto list = DCast<Model::List>(node->parent()))
 			{
+				for (auto view : target->scene()->viewItems()->viewItemsAsList())
```

Comment:
I also put it in Item::removeFromScene.

So the reason I put something here as well is as follows: If a node is deleted, then the corresponding InfoNodes should be deleted. However, in the Viewitem, the same node might be visualized by many different items. Therefore, doing this when removing the item itself might be a too aggressive way to remove them (the InfoNode could refer to another visualization of the same node). Any ideas how to solve this?

I put code in Item::removeFromScene as well.